### ISSUE TITLE
Is this project inactive?

### ISSUE THREAD

**Issue by EDKarlsson** [Wiredcraft/loopback-connector-couchbase3#142]
I see that this is the most recent available couchbase connector for loopback. Viewing the orgs projects it looks like you moved away from loopback while sticking with couchbase. Would it be possible to know why this decision was made? Currently looking into loopback and Couchbase.

**Comment by xavierchow** [comment]
> moved away from loopback while sticking with couchbase. 

right, we moved to https://nestjs.com/ as most of team members are comfortable with it and it has much more open-source libraries around it's eco-system.

@EDKarlsson

**Comment by EDKarlsson** [comment]
Thanks for getting back to me, I really appreciate it! 

Ah okay. Was the loopback framework learning curve difficult? I find the documentation sparse and there's a lot of abstraction to the code. I'm also looking to use a time series database as well; I am curious to know how much work it is to develop and maintain a connector? I'm not too familiar with JS API frameworks but I usually judge a framework by its documentation. NestJS seems to be well documented compared to Loopback.

**Comment by xavierchow** [comment]
@EDKarlsson 
The learning curve is hard to say, but you are right there are quite a lot of abstractions, if you want a connector for loopback, you have to follow its interfaces. I'm not quite sure about loopback 4 cuz we just implemented this library for loopback 3, and we moved away from loopback.

You didn't mention what database you're gonna use, if you have a corresponding JS or TS SDK for the DB it would be good enough to start with, you don't have to use a `connector`.
However, if you are looking for a more ORM / ODM thing then it's another topic.

**Comment by EDKarlsson** [comment]
My team is looking at different ones. At the moment we are using react with mongodb. I had the task to explore the alternatives. We are strongly considering couchbase and influxdb. We need a document store and a time series db. Other ones we are considering are postgresql and timescaledb. We know mongodb can fulfill both document and time series but a lot of the features we need aren't available for on premise deployment.
